source/_integrations/spotify.markdown

```diff
@@ -113,6 +115,32 @@ script:
 
 The `media_content_id` value can be obtained from the Spotify desktop app by clicking on the more options ("...") next to the album art picture, selecting "Share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
 
+The `media_content_id` can be a URL (e.g. `https://open.spotify.com/playlist/37i9dQZF1DWVFeEut75IAL`) or the Spotify URI string (e.g. `spotify:playlist:37i9dQZF1DWVFeEut75IAL`). (play_media)[https://www.home-assistant.io/integrations/media_player/#service-media_playerplay_media] requires the correct corresponding `media_content_type`.
+
+## Selecting output source
+
+To play media Spotify needs a device selected for audio output known as the `source`. Currently, this integration cannot initiate playback to a device not already available from the Spotify API. You need to first use Spotify on your smart speaker or phone to populate the source list.
+
+```yaml
+# Example code to select an AV receiver as the output device
+service: media_player.select_source
+entity_id: media_player.spotify
+data:
+  source: "Denon AVR-X2000"
+```
+
+The device name must be known to Spotify and therefore is often different from names given to entities in Home Assistant.
+
+The source list of available devices can be found in the Details section of the Spotify Media Player Control and the `source_list` attribute in the Developer tools States or templates using:
+
+```jinja
+{{ state_attr('media_player.spotify', 'source_list') }}
+```
+
+<div class='note'>
+  A known limitation with Spotify is that devices disappear from the source list over time. Many Spotify Connect devices tend to persist, other devices such as Amazon Alexa speakers may only be remembered for a while, and phones and Google Cast devices disappear as soon as playback stops.
```

Comment:
Let's change this to something more concrete instead of "magically disappearing".

e.g., 

> Spotify Connect devices are only visible in the list of media sources when they are actively used with the connected Spotify account.

Reply:
I like the positive wording you here. I will drop the vague reference to particular brands as it is hard to verify.

Of those that do persist (not phones or Google Cast) it seems unclear why devices do "magically disappear". I think someone said Spotify only recalls the last 3 connect devices but I cannot find this documented anywhere